hack/feature-deploy.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
 
+export KUSTOMIZE_DIR="${KUSTOMIZE_DIR:-/tmp}"
+export KUSTOMIZE_BIN=$KUSTOMIZE_DIR/kustomize
+
+# Function: download the latest Kustomize binary to /tmp
+function get_kustomize_binary (){
+    pushd /tmp
+    curl -s "https://raw.githubusercontent.com/\
```

Comment:
I think we should add a
```
-m 600
```
so that curl will never hang

Reply:
BTW: as a result of this review
I've opened https://github.com/kubernetes-sigs/kustomize/pull/3357
so the downloaded script here will also behave nicely

IMHO 10minutes is a sane timeout